core/services/chainlink/application.go

```diff
@@ -1412,73 +1414,153 @@ func newCCVServices(
 		var services []services.ServiceCtx
 		globalLogger = globalLogger.Named("CCV")
 
-		// TODO: move config from hardCodedCCVConfig into general config.
-		waiting := true
-		for waiting {
-			globalLogger.Infof("Waiting, use debugger to continue...")
-			time.Sleep(1 * time.Second)
-			waiting = waiting
-		}
-
 		legacyRelayers := getLegacyChains(globalLogger, relayerChainInterops)
 		ccvCfg, err := ccvConfig(1)
 		if err != nil {
-			globalLogger.Errorf("Failed to get CCV config: %v", err)
+			globalLogger.Errorf("[CCVSTARTUP] Failed to get CCV config: %v", err)
+		}
+
+		// hacky way to get all the signer keys from all the verifier configs.
+		// this won't be needed when we switch to the ocr2 keystore.
+		var allSignerKeys []ethkey.KeyV2
+		for _, i := range []int{1, 2, 3} {
+			ccvCfg, err := ccvConfig(i)
```

Comment:
I was initially thinking we would have 3 nodes. Rather than adding a second layer of hacks, `ccvConfig()` should be modified to return everything.